### ISSUE TITLE
Native Pro Controller Support

### ISSUE THREAD

**Issue by AE-GM** [moonlight-stream/moonlight-qt#1040]
**Is your feature request related to a problem? Please describe.**
Yes, when I use either xbox or ps4, the capture button and the gyroscope/motion controls aren't working and aren't registered

**Describe the solution you'd like**
Providing a full pro controller or joycons controls support for emulation purpose

**Describe alternatives you've considered**
Detecting the type of controller through bluetooth name and then providing the gamepad input through that

**Additional context**
basically the motion controls and capture button doesn't work when connected through moonlight (on tv), but the controller works fine when connected directly to the pc through bluetooth

**Comment by cgutman** [comment]
This already implemented in 9af58af5e4aa536083ad62997135c1f9c03430ec and currently requires Sunshine nightly builds (until v0.21).

**Comment by AE-GM** [comment]
at least tell me how to get it working

**Comment by AE-GM** [comment]
oh I posted in the wrong repo
